src/revivalpmmp/pureentities/PureEntities.php

```diff
@@ -486,4 +496,4 @@ public static function getPositionNearPlayer(Player $player, int $minimumDistanc
 
 		return new Position($x, $player->y, $z, $player->getLevel());
 	}
-}
\ No newline at end of file
+}
```

Comment:
no ending lines

Reply:
I'm editing online in the code editor, for the life of me I can't remove these (nor did I add them)?

Reply:
Confirmed, no empty new line at end of file?